### PR TITLE
Braginskiiwarning

### PR DESCRIPTION
**Pull request by tlee0818**
fixed units warnining in validators

### CODE REVIEW

**Comment by tlee0818** [comment]
fixed the linter issues. no need for a new pr.

**Comment by tlee0818** [comment]
this also suppresses warnings for collisions.py

**Comment by tlee0818** [comment]
also suppresses warnings for formulary/tests/test_transport.py

File: plasmapy/utils/decorators/validators.py [hunk 1]
```diff
@@ -58,6 +58,7 @@ class ValidateQuantities(CheckUnits, CheckValues):
         can_be_nan             `bool`  [DEFAULT `True`] values can be :data:`~numpy.nan`
         none_shall_pass        `bool`  [DEFAULT `False`] values can be a python `None`
         can_be_zero            `bool`  [DEFAULT `True`] values can be zero
+        can_be_zero            `bool`  [DEFAULT `True`] values can be zero
```
Comment (andrewsheng2) [inline plasmapy/utils/decorators/validators.py:61]: This comment seems to be wrong since it is a duplicate of the above line

**Pull request merged**